CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Changed
 
+-   Fixed styling conflict in `InputText` introduced by Addon element
```

Comment:
This should go under a `### Fixed` header. Also use present tense for the new entry 🙏🏻
Have a look at the [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) guidelines.